### DISCUSSION TITLE
scrum prep Milestone 3- 4

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#206] [General]
Write your scrums here

**Comment by MrL9000** [comment]
Updated the main menu so that it shows the user's name instead of the placeholder "user". Decided to do this task over the other 2 I have due to it's uncertanty in length. Will do these in the comming weekend.

Edit: accidentally refferenced the task I did last sprint

**Comment by Yahid1** [comment]
Prepared the official website of the Admin Dashboard to be publicly accessible. Discussed with the main RUMBus Team to determine user issues and bugs that need to be solved.

**Comment by JaseyAcevedo** [comment]
Working on ltt and projects for the 27th

**Comment by Janiel777** [comment]
Estoy trabajando en dos LTT pero está semana no hice muchos avances.

**Comment by gabrielpadilla7** [comment]
Working on creating new issues and closing/reviewing issues

**Comment by IvanierB** [comment]
Working on two ltt to complete this milestone, should be finished on the weekend.

**Comment by rafa-quiles** [comment]
Finishing some documentation tasks

**Comment by AleDiazz** [comment]
finished some documentation task and working on my last ltt

**Comment by diegorobles2** [comment]
Been working on my last ltt and tasks

**Comment by Brandell13** [comment]
I was working to create new tasks

**Comment by sebasagir** [comment]
been working on ltt's

**Comment by juan-ortiz92** [comment]
This week I started wroking on my final lecture topic task.

**Comment by marcosarraiza** [comment]
This week i started work on a lecture tooic task and divided my work with a peer on another lecture topic task

**Comment by Alej24** [comment]
I have been investigating how to do a LTT, but I have been quite busy so I haven't been able to do anything else.

**Comment by dAninAd1248** [comment]
Nothing, was busy with tests.

**Comment by yamayrie** [comment]
worked on a lecture topic task and fixing documentation

**Comment by AlexisTraversoPR** [comment]
Was not able to do much this week

**Comment by JConcepcion01** [comment]
Didnt work on much this week since I had done my major tasks last week

**Comment by JoelTorres20** [comment]
I wasn't able to do much during the week, but I was able work on my tasks over the weekend
